spec/cookbook_spec.rb

```diff
@@ -170,18 +180,52 @@
       :result => [
       ],
     },
+    {
+      :name => 'when metadata file is not in the root of the cb dir',
+      :files => [
+        {
+          :status => :deleted,
+          :path => 'cookbooks/two/cb_one/files/default/metadata.rb',
+        },
+      ],
+      :result => [
+        ['cb_one', :modified],
+      ],
+    },
   ]
 
-  fixtures.each do |fixture|
-    it "should handle #{fixture[:name]}" do
-      BetweenMeals::Changes::Cookbook.find(
-        fixture[:files],
-        cookbook_dirs,
-        logger,
-      ).map do |cb|
-        [cb.name, cb.status]
-      end.
-        should eq(fixture[:result])
+  {
+    'Normally' => false,
+    'With symlinks' => true,
+  }.each do |c, track_symlinks|
+    context "Running BetweenMeals #{c}" do
+      before do
+        allow(File).to receive(:realpath).with(repo_path).and_return(repo_path)
+        if track_symlinks
+          cookbook_dirs.each do |dir|
+            repo = File.join(repo_path, dir)
```

Comment:
It's not clear this properly tests your code since fixture[:files] still includes all the same thing. You'd need a different test where fixture[:files] is in some OTHER dir, but you have a symlink to them inside the cookbook dir, right?